mtags/src/main/scala/scala/meta/internal/mtags/ClasspathLoader.scala

```diff
@@ -77,6 +78,10 @@ final class ClasspathLoader() {
     loader.resolve(path)
   }
 
+  def loadClass(symbol: String): Option[Class[_]] = {
+    Try(loader.loadClass(symbol)).toOption
```

Comment:
I would do logging if error happens. It will help investigate if something will start failing.

Reply:
We do a lot of guessing here, some weirdly defined classes or cases where we will never find the symbol will just create additional noise.

Reply:
Added onError, which will only do `scribe.debug`, which should not add noise unless the user turns on the debug flag.